Add webview support

jnugh opened the pull request:
Hey there,
I have been trying to get `electron-context-menu` to work in an environment with multiple WebViews.

Turns out the `context-menu` event does not propagate through the application when it gets fired in the webviews `webContents`.

I tried to attach the event handle manually by passing a window param to the create method, but `webView.webContents` does not exist, instead one has to use `webView.getWebContents()`.

Plus it's not possible to open the menu on a webview, so I changed menu.popup to use the currently focused window which makes sense, I guess.

sindresorhus commented:
Can you add a webview to https://github.com/sindresorhus/electron-context-menu/blob/master/fixture.html so I can manually verify it works?

sindresorhus commented on readme.md:
```diff
@@ -45,9 +45,9 @@ app.on('ready', () => {
 #### window
 
-Type: `BrowserWindow`<br>
+Type: `BrowserWindow` / `WebView`<br>
```
Drop the `/`

jnugh commented:
I added an example to `fixture.html`. With which I found, that `webview`s webContents is undefined during load. Therefore I added a `deferedCreate` method which waits for `dom-ready`.

sindresorhus commented:
> With which I found, that webviews webContents is undefined during load. Therefore I added a deferedCreate method which waits for dom-ready.

The Electron docs say nothing about that. Sounds like an Electron bug. Can you open an issue on Electron?

sindresorhus commented on index.js:
```diff
@@ -92,14 +92,36 @@ function create(win, opts) {
+		const webContents = win.webContents || win.getWebContents();
+
+		if (webContents === undefined) {
+			return deferedCreate(win, opts);
```
Just put `deferedCreate` inline here. No need for a function. And use `on` instead of `addEventListener`.

sindresorhus commented on index.js:
```diff
@@ -92,14 +92,36 @@ function create(win, opts) {
 	if (opts.window) {
-		create(opts.window, opts);
-		return;
+		// When window is a webview that has not yet finished loading webCOntents is not available
```
typo

sindresorhus replied:
And comment is placed out of context

sindresorhus commented on readme.md:
```diff
@@ -45,9 +45,9 @@ app.on('ready', () => {
 #### window
 
-Type: `BrowserWindow`<br>
+Type: `BrowserWindow WebView`<br>
```
Should be in separate code blocks.

sindresorhus commented on fixture.html:
```diff
@@ -16,5 +24,6 @@
 		<div contenteditable="true"><strong>Editable text</strong></div>
 		<br>
 		<a href="https://github.com">Link</a>
+		<webview id="webview" src="http://example.org" />
```
```html
<webview id="webview" src="http://example.org">
```

jnugh commented:
Fixed the code style issues mentioned above & created https://github.com/electron/electron/issues/6920

jnugh commented:
Got an answer pretty fast. 
Didn't see this in the docs as well:

> Note: The webview element must be loaded before using the methods.

sindresorhus commented:
> Didn't see this in the docs as well:

I missed that as well.
